server/services/twitter.js

```diff
@@ -0,0 +1,62 @@
+import superagent from 'superagent';
+import { createTweet } from '../model/tweet';
+
+async function getTwitterAccessToken() {
+  // TODO: reimplement after testing
+  const urlEncodedK = encodeURIComponent('qpsVJlsTg22FFKr1fwzkzU4Ew');
+  const urlEncodedS = encodeURIComponent('kmXPnluTYkl7oH4mRlC1EgthH8KLT9mBpezgK3wBaFQviC23Ru');
+  const base64EncodedKS = Buffer.from(`${urlEncodedK}:${urlEncodedS}`).toString('base64');
+
+  return new Promise((resolve, reject) => {
```

Comment:
Do we need to use this?  We could use `async` `await` as `superagent` is already Promisified.